### PR TITLE
Remove CONFIG_NF_NAT_IPV4 check

### PR DESCRIPTION
**Pull request by bernardjkim**
## Description

Removes the `CONFIG_NF_NAT_IPV4` check. Needed for CentOS/RHEL 8.3 support.

## Type of change

* Internal change (not necessarily a bug fix or a new feature)

## Linked tickets and other PRs

* Refs https://github.com/gravitational/gravity/issues/2331

* Requires https://github.com/gravitational/planet/pull/795, https://github.com/gravitational/satellite/pull/285

## TODOs

- [x] Perform manual testing
- [x] Address review feedback
- [x] Update upstream references / tags / versions after upstream PR merges (linked above)

## Testing done

- [x] Gravity installs on CentOS/RHEL 8.3

### CODE REVIEW

**Comment by wadells** [comment]
I approve in spirit.  Don't we need the final hashes from after the merges before this can merge though?

**Comment by bernardjkim** [comment]
Yup, I'll update once Satellite and Planet PRs are merged.

**Review by wadells — APPROVED** [review]
Good to merge once the final hashes are in!

**Review by knisbet — APPROVED** [review]

**Pull request merged**
